utils.py

```diff
@@ -96,4 +96,13 @@ def wrapped_func(*args,**kwargs):
         return func(d)
     return wrapped_func
 
+#Skip redundent expressions like repeats on rarity, properties, etc.
+def redundant(word): 
```

Comment:
I didn't test well enough to see how silly the descriptions look when we don't remove redundant lines. Lets use this function during the display function, I still want to keep everything in self.description but we don't have to print it.